Safor/index.html

```diff
@@ -0,0 +1,160 @@
+                        <h2 class="four_list-item_headline four_list-item_headline_third">Anonymity</h2>
+                        <p>Your relatives can be notified that you need help only with your consent</p>
+                    </li>
+                </ul>
+            </div>
+        </section>
+        <section class="five">
+            <div class="five_one">
+                <h2 class="five_one_headline">FOR BUSINESS</h2>
+                <ul>
+                    <li>
+                        <img src="img/five/Rectangle_6.png">
+                        <p>At the heart of our activities is an immediate response to incoming 
+                            proposals from our clients and partners, changing external and internal
+                            factors and signals for help.</p>
+                    </li>
+                    <li>
+                        <img src="img/five/Rectangle_6_Copy.png">
+                        <p>Business proposals, external and internal requests and new technologies.
+                            Transparency of all processes of interaction with our employees, customers 
+                            and business partners.</p>
+                    </li>
+                    <li>
+                        <img src="img/five/Polygon_Copy.png">
+                        <p>In making decisions, the quality of the services provided and the distribution 
+                            of funds. We use all available resources to achieve maximum results in 
+                            our work and communication.</p>
+                    </li>
+                </ul>
+            </div>
+            <div class="five_two">
+                <h2 class="five_two_headline">FOR PEOPLE</h2>
+                <ul>
+                    <li>
+                        <img src="img/five/Polygon.png">
+                        <p>Do not waste time to explain your location and nature of the threat, guard 
+                            patrols get your alarm in 1 second and advances to you no matter whether you 
+                            are in motion or not.</p>
+                    </li>
+                    <li>
+                        <img src="img/five/Star.png">
+                        <p>You will be foundregardless of whether you are in town or on a trip.
+                            The service covers the entire territory of the country and allows you to specify
+                            the exact location even when in a crowded place</p>
+                    </li>
+                    <li>
+                        <img src="img/five/Rectangle_6_Copy_2.png">
+                        <p>And we will help in any situation. Only after coming to you the calling
+                            will be closed by the patrol arrived, whose goal is to protect and help you.</p>                    
+                    </li>
+                </ul>
+            </div>
+        </section>
+        <section class="six">
```

Comment:
Change names of sections to be more descriptive, for example: footer-feedback or feedback-form.
Please do this for every section and sub div in the document.